test/cmdLineTests/shareClassTests/SCCMLTests/ShareClassesCMLTests-2.xml

```diff
@@ -0,0 +1,791 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+
+<!--
+  Copyright (c) 2012, 2018 IBM Corp. and others
+
+  This program and the accompanying materials are made available under
+  the terms of the Eclipse Public License 2.0 which accompanies this
+  distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+  or the Apache License, Version 2.0 which accompanies this distribution and
+  is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+  This Source Code may also be made available under the following
+  Secondary Licenses when the conditions for such availability set
+  forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+  General Public License, version 2 with the GNU Classpath
+  Exception [1] and GNU General Public License, version 2 with the
+  OpenJDK Assembly Exception [2].
+
+  [1] https://www.gnu.org/software/classpath/license.html
+  [2] http://openjdk.java.net/legal/assembly-exception.html
+
+  SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+<!DOCTYPE suite SYSTEM "cmdlinetester.dtd">
+
+<!-- Test 1 to Test 52: 68 tests -->
+
+<!-- #### RUN TokenHelperTests SUITE #### -->
+<suite id="Shared Classes CommandLineOptionTests Suite - Windows Configuration">
+
```

Comment:
Can " - Windows Configuration" be removed ?

Reply:
Should be.

Reply:
Or could be leave as is. Those window setup are related with tests with cacheDirPerm sub option, which are in ShareClassesCMLTests-4(5). Together with other variable setting and tests "Start : Cleanup: persistent" and test "Start: Cleanup: nonpersistent" are the common beginning part of all those ShareClassesCMLTests-1//2/3/4/5/6.xml files.

Reply:
I don't know what "Windows Configuration" means here. It gives a false impression that the following variables are for Windows only. My preference is to remove it. But I will leave it to you.